moped-editor/src/views/projects/projectView/ProjectComponents/mapStyleSettings.js

```diff
@@ -211,20 +248,17 @@ export const MAP_STYLES = {
       type: "circle",
       _featureIdProp: "INTERSECTION_ID",
       paint: {
-        "circle-radius": {
-          stops: [
-            [5, 4],
-            [16, 10],
-          ],
-        },
-        "circle-stroke-opacity": 0.9,
```

Comment:
This was causing the code to grab the wrong layer ID from a small clickable area. I updated the circle styles to be uniform across all the points layers.